flocker/control/script.py

```diff
@@ -77,6 +81,35 @@ def main(self, reactor, options):
 
 
 def flocker_control_main():
+    pr = cProfile.Profile()
```

Comment:
Bonus comment: Use `Profile(time.clock)` to get cpu-time instead of wallclock time.